internal/cmd/shim.go

```diff
@@ -69,6 +71,42 @@ func pluginPythonCode(s config.SQLPython) *plugin.PythonCode {
 	}
 }
 
+func pluginGoCode(s config.SQLGo) *plugin.GoCode {
+	return &plugin.GoCode{
+		EmitInterface:             s.EmitInterface,
+		EmitJsonTags:              s.EmitJSONTags,
```

Comment:
unfortunately, i don't think this name mismatch can be resolved by coaxing protoc-gen-go: https://github.com/golang/protobuf/issues/555

we could alternatively rename the native go structs or try to get rid of this indirection altogether.